docs/docs/cmd/tenant/people/people-profilecardproperty-add.mdx

```diff
@@ -0,0 +1,171 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# tenant people profilecardproperty add
+
+Adds a custom attribute as a profile card property
```

Comment:
```suggestion
Adds an additional attribute to the user profile card properties
```

Reply:
it's not called user profile card in the docs, so I'll leave user out.